tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/date_and_time.py

```diff
@@ -10,6 +10,7 @@
 
 
 class DateAndTime(Base):
+    _screen_locator = (By.ID, 'dateTime')
```

Comment:
actually, come to think of it, I think _page_locator is a better name since it's the reference for that page, not a screen view.  I'll change all _screen_locator to _page_locator